nightly-style-policy.md

```diff
@@ -0,0 +1,26 @@
+Initial PR(s) implementing new syntax filed against rust-lang/rust should
+generally leave the rustfmt subtree untouched. In cases that that need to
+modify rustfmt (for example, to fix compiler errors that come from adding
+new variants to AST representation), they should modify rustfmt in such a
+way to keep existing formatting verbatim.
+
+Rustfmt is allowed to implement nightly-only formatting behavior without that
+syntax being specified by the style guide. Initial PR implementors are 
+encouraged but not required to open a PR against rustfmt suggesting an initial
+formatting behavior, or formatting may later be implemented as a PR by anyone,
+pending approval of the implementation from T-rustfmt. T-style should be
+notified to approve the interim style proposed by these PRs, but this decision
+is not binding and may be revisited until the feature is stabilized and the
+formatting is codified in the style guide. 
+
+Much like breaking nightly feature changes in the Rust compiler, changes should
+be not done unnecessarily, and should take into account the feature's adoption
+and readiness for stabilization. However, changes may be done until the feature
```

Comment:
Another one I don't feel strongly about, just food for thought with somewhat wordier phrasing to expand upon motivation and implications around a threshold for change "necessity"

```suggestion
Much like breaking nightly feature changes in the Rust compiler, any changes
to formatting behavior for nightly syntax should be made cautiously and with
thorough consideration to avoid churn. Changes should not be done unnecessarily,
and should take into account the feature's adoption
and readiness for stabilization. However, changes may be done until the feature
```